src/main/webapp/app/controllers/submission/submissionListController.js

```diff
@@ -29,6 +29,8 @@ vireo.controller("SubmissionListController", function (NgTableParams, $controlle
 
     $scope.fieldPredicates = FieldPredicateRepo.getAll();
 
+    var rowFilterTitle = "Exclude";
```

Comment:
Do we need this?

Reply:
We already do this in the pre-existing code.
We use the same string in multiple places now so I changed the scope appropriately.